bucket/crc.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "1.8.0",
+    "description": "Manages a local OpenShift 4.x cluster optimized for testing and development purposes.",
+    "homepage": "https://code-ready.github.io/crc/",
+    "license": "Apache-2.0",
+    "notes": "To acquire a pull secret, visit: https://cloud.redhat.com/openshift/install/crc/installer-provisioned",
+    "architecture": {
+        "64bit": {
+            "url": "https://mirror.openshift.com/pub/openshift-v4/clients/crc/1.8.0/crc-windows-amd64.zip",
+            "hash": "a6e87ef46699d5d8da1e3960818a57b53f329ea9a190d40ec025371f23547183",
+            "extract_dir": "crc-windows-1.8.0-amd64"
+        }
+    },
+    "bin": "crc.exe",
+    "checkver": {
+        "github": "https://github.com/code-ready/crc"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://mirror.openshift.com/pub/openshift-v4/clients/crc/$version/crc-windows-amd64.zip",
+                "extract_dir": "crc-windows-$version-amd64",
+                "hash": {
+                    "url": "$baseurl/sha256sum.txt"
+                }
+            }
+        }
```

Comment:
```suggestion
                "extract_dir": "crc-windows-$version-amd64"
            }
        },
        "hash": {
            "url": "$baseurl/sha256sum.txt"
        }
```

Reply:
Please explain the requested changes

Reply:
What You do not understand?

This hash file is generic. It should be outside of architecture.

Reply:
Sincerely grateful for clarification!